acquire/acquire.py

```diff
@@ -2166,6 +2166,21 @@ def main() -> None:
     except ValueError as err:
         parser.exit(err)
 
+    if args.targetd:
+        from targetd.tools.targetd import start_client
+
+        # Configure your targetd agent here
+        config = {
+            "function": "agent",
+            "topics": ["/host/a", "/host/*"],
+            "link": "pipe-server://tmp/b",
+            "address": "10.0.2.3",
```

Comment:
I'm not sure of the purpose of this PR, but maybe best not to merge it with test data?

Reply:
This is kind of an intermediate version (hardcoded). Full version with integrated config will follow today.

Reply:
Config is now dynamically injected conform procedure in https://github.com/fox-it/acquire/pull/105/commits/f3b924ff8ccc3f5cb866901e4c2415a2614c5b89.